ts-sdk/whirlpool/src/config.ts

```diff
@@ -10,18 +10,18 @@ export const DEFAULT_ADDRESS = address("11111111111111111111111111111111");
 /**
  * The WhirlpoolsConfig addresses for various networks.
  */
-export const WHIRLPOOLS_CONFIG_ADDRESSES = {
-  SolanaMainnet: address("2LecshUwdy9xi7meFgHtFJQNSKk4KdTrcpvaB56dP2NQ"),
-  SolanaDevnet: address("FcrweFY1G9HJAHG5inkGB6pKg1HZ6x9UC2WioAfWrGkR"),
-  EclipseMainnet: address("FVG4oDbGv16hqTUbovjyGmtYikn6UBEnazz6RVDMEFwv"),
-  EclipseTestnet: address("FPydDjRdZu9sT7HVd6ANhfjh85KLq21Pefr5YWWMRPFp"),
+export const DEFAULT_WHIRLPOOLS_CONFIG_ADDRESSES = {
+  solanaMainnet: address("2LecshUwdy9xi7meFgHtFJQNSKk4KdTrcpvaB56dP2NQ"),
```

Comment:
should we just call it `solana` and `eclipse` without mainnet?

Reply:
Personally, I prefer with mainnet, because it's more descriptive? You have the final say.